docs/cudf/source/user_guide/performance_comparisons.ipynb

```diff
@@ -0,0 +1,1568 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    pdf = pdf.head(100_000_000)
</pre><p>Why did we create 300M rows, only to trim it to 100M? It's a bit confusing to define <code>num_rows</code> and then not benchmark data with that number of rows. Let's use 100M or 300M everywhere.</p><p> </p><p>I see below that we redefine <code>num_rows = 1_000_000</code> with a new dataset. That's okay - but within the benchmarks of a given dataset, we should be consistent so that it doesn't look like we're cherry-picking data sizes to show the best speedup (especially for smaller datasets like 1M or 100M instead of 300M).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cudf/pull/12595/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>